Terning-iOS/Terning-iOS/Source/Presentation/NewHome/Cell/CheckDeadlineCell.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  CheckDeadlineCell.swift
+//  Terning-iOS
+//
+//  Created by 김민성 on 9/9/24.
+//
+
+import UIKit
+
+import SnapKit
+import Then
+
+final class CheckDeadlineCell: UICollectionViewCell {
+    
+    // MARK: - UIComponents
+    
+    private let checkDeadlineCard = UIView().then {
+        $0.makeBorder(width: 1, color: .grey150, cornerRadius: 5)
+        $0.backgroundColor =  .white
+        $0.layer.applyShadow(color: .black, alpha: 0.25, x: 0, y: 0, blur: 4, spread: 0)
+        $0.clipsToBounds = true
+    }
+    
+    private let checkDeadlineCardLabel = LabelFactory.build(
+        text: "일주일 내에 마감인 공고가 없어요 \n 캘린더에서 스크랩한 공고 일정을 확인해 보세요",
+        font: .detail2,
+        textColor: .grey400
+    ).then {
+        $0.numberOfLines = 2
+    }
+    
+    lazy private var checkDeadlineButton = UIButton().then {
```

Comment:
```suggestion
    private lazy var checkDeadlineButton = UIButton().then {
```
문제 없는 코드긴 한데, 보통 접근 제어자를 가장 앞에 써주는 형식을 많이 사용합니다!

Reply:
아 그러네욥 바로 수정했습니다!

Reply:
` lazy private var `이렇게 써도 작동되는거 지금 알았네 ㅋㅋ